tests/integration/test_team_view.py

```diff
@@ -98,19 +98,20 @@ def test_change_group_non_member(self, client, user, team):
         group2 = Group(name='group2')
         group2.save()
         client.force_login(user)
-        response = client.patch(f'/team/{team.public_identifier}/', {'group': 'group2'},
+        response = client.patch(f'/teams/{team.public_identifier}/', {'group': 'group2'},
                                 content_type='application/json')
         assert response.status_code == 403
 
     def test_change_group_not_exists(self, client, logged_in_user, team):
-        response = client.patch(f'/team/{team.public_identifier}/', {'group': 'group2'},
+        response = client.patch(f'/teams/{team.public_identifier}/', {'group': 'group2'},
                                 content_type='application/json')
         assert response.status_code == 404
 
     def test_create(self, client, logged_in_user, team, group):
+        assert models.Team.objects.count() == 2
```

Comment:
Perhaps make no assumptions on what is in the database but just keep track of the count and on line 117 verify that the new count is +1? That way it is not needed to update these values in the tests whenever a new team is added